### PR TITLE
Spelling improvements for dmsguide, monstermanual, and rulesdoc

### PR DESCRIPTION
**Pull request by mel-riv**
Did some basic spellchecking for the .md files.

### CODE REVIEW

File: dmsguide.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ This lantern has enough oil to light for 8 hours. When lit it acts as a normal l
 #### Cloak of Agility
 *Wondrous Item, Requires Attunement*
 
-This cloak grants you +1 to your Dexterity. If your movespeed is lower than your sprint speed, it is increased to your sprint speed.
+This cloak grants you +1 to your Dexterity. If your move speed is lower than your sprint speed, it is increased to your sprint speed.
```
Comment (mel-riv) [inline dmsguide.md:248]: changing to "move speed" vs "movespeed" for consistency

File: rulesdoc.md [hunk 2]
```diff
@@ -622,7 +622,7 @@ Many GM's will represent their world with a gameboard, with every tile on the bo
 | Medium | 1 |
 | Large | 2x2 |
 | Huge | 3x3 |
-| Gargantuant | 4x4+ |
+| Gargantuan | 4x4+ |
```
Comment (mel-riv) [inline rulesdoc.md:647]: Verify this please.

File: rulesdoc.md [hunk 3]
```diff
@@ -1009,11 +1009,11 @@ You can maintain flight for a limited time. On your turn, you may take flight us
 - [Power Word: Lightning](#Power-Word-Lightning)
 - [Project Image](#Project-Image)
 
-##### Eigth
+##### Eighth
 - [Fiendfire](#Fiendfire)
 - [Invulnerability](#Invulnerability)
 - [Slow Time](#Slow-Time)
-- [Stablize Warren](#Stablize-Warren)
+- [Stabilize Warren](#Stabilize-Warren)
```
Comment (mel-riv) [inline rulesdoc.md:1078]: Unsure if you'll have to update roll20 if the links/titles change

File: rulesdoc.md [hunk 4]
```diff
@@ -2454,7 +2454,7 @@ ___
 
 You change the aura of a creature or object, changing how it appears to magical effects, such as _Detect Magic_ or _Identify_. Target up to 5 creatures or objects, adding a bad die to the spellcasting roll for every target. On success, you may do any of the following:
 
-You can cause a mundane object to appear magical, a magical object appear mundane, or change it's magical aura to belong to a different school. For example, you could cause an arcane effect to appear as divine when seen by _Detect Magic_. You can change how a creature appears to different spells, such as disguising an Undead creature as a Human for the effects of _Concencrate Ground_. You may change how their race, type, or size appears to spells.
+You can cause a mundane object to appear magical, a magical object appear mundane, or change it's magical aura to belong to a different school. For example, you could cause an arcane effect to appear as divine when seen by _Detect Magic_. You can change how a creature appears to different spells, such as disguising an Undead creature as a Human for the effects of _Consecrate Ground_. You may change how their race, type, or size appears to spells.
```
Comment (mel-riv) [inline rulesdoc.md:2519]: verify this please

File: rulesdoc.md [hunk 5]
```diff
@@ -2910,7 +2910,7 @@ You may choose to spend a minute casting this spell instead. If you do so the se
 - [Bramble](#Bramble)
 - [Breath](#Breath)
 - [Chilling Fog](#Chilling-Fog)
-- [Close Wounds](#Close-Wounds)
+- [Close Wound](#Close-Wound)
```
Comment (mel-riv) [inline rulesdoc.md]: This one has been bothering me. Is it Close Wound, or Close Wound**s**?

File: rulesdoc.md (hunk 4 above)
Comment (scrabbletank) [reply rulesdoc.md:2519]: thats correct!

File: rulesdoc.md (hunk 5 above)
Comment (scrabbletank) [reply rulesdoc.md]: I think it should be Wounds as it closes multiple.. sometimes

File: rulesdoc.md (hunk 2 above)
Comment (scrabbletank) [reply rulesdoc.md:647]: correct!

File: rulesdoc.md (hunk 3 above)
Comment (scrabbletank) [reply rulesdoc.md:1078]: yeah, but thats basically just running a script, I can check that in after.

File: dmsguide.md [hunk 6]
```diff
@@ -602,22 +602,22 @@ A tunic woven from the hair of a spriggan. While outside you have expertise in s
 - **AP Cost:** 5
 - **Range:** 7
 
-When standing next to a tree you may magically enter the tree. While inside the tree you are untargetable by effects that do not target the ethereal plane. At the start of your next turn you appear beside a tree within range.
+When standing next to a tree you may magically enter the tree. While inside the tree you are un-targetable by effects that do not target the ethereal plane. At the start of your next turn you appear beside a tree within range.
```
Comment (scrabbletank) [inline dmsguide.md]: Everything in these changes are great, although we probably need to change all that 'un-'s back to 'un'. Untargetable is a valid word, at least in the gaming world. Even though it has big red squigglies as I'm writing this.

**Review by scrabbletank — COMMENTED** [review]
need to change un- to un, but thanks! That is way more spelling mistakes than I thought.

**Review by scrabbletank — APPROVED** [review]

**Pull request merged**